README.md

```
@@ -1,16 +1,68 @@
-# EduAid: AI Quiz Generation 🚀
+<h1 align="center">EduAid 🎓 | AI-Powered Quiz Generator</h1>
 
-Online learning has taken the front seat in the post-pandemic age. With the advent of sophisticated AI architectures like Transformers, it is only natural that AI would find its way into education. Learning online via platforms like YouTube or MOOCs is often a method of self-learning. The biggest obstacle faced by students in self-learning is the lack of attention span. An online tool that can generate short quizzes from input educational content can be a great resource for both teachers and students. It helps retain important information, frame questions, and quickly revise large chunks of content.
+<p align="center">
+  <img src="https://img.shields.io/badge/GSoC-2025-orange?style=for-the-badge" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)